server/app/services/kommission-service.js

```diff
@@ -0,0 +1,101 @@
+const logger = require('../../logger');
+const { Kommission: Kommission } = require('../models/kommission');
+const { createUserError } = require('../../utils');
+
+/**
+ * Return all kommissions of the app.
+ *
+ * @returns {Promise<Array>} Kommissions
+ */
+async function getAllKommissions() {
+
+    logger.verbose('Kommission service: get all kommissions');
+    return await Kommission.findAll();
+}
+
+
+/**
+ * Create a kommission.
+ *
+ * @param newKommission {Kommission} partial kommission
+ * @return {Promise<Kommission|Errors.ValidationError>} The created kommission with its id
+ */
+async function createKommission(newKommission) {
+    
+    logger.verbose('Kommission service: creating a new kommission named %s %s', newKommission.firstName, newKommission.lastName);
+    return await newKommission.save();
+}
+    
+
+/**
+     * Get a kommission by its id.
+     *
+     * @param kommissionId {number} Kommission id
+     * @return {Promise<Kommission>} The wanted kommission.
+     */
+async function getKommissionById(kommissionId) {
+    
+    logger.verbose('Kommission service: get kommission by id %d', kommissionId);
+    
+    const kommission = await Kommission.findById(kommissionId);
+    
+    if (!kommission) throw createUserError('UnknownKommission', 'This kommission does not exist');
```

Comment:
We will need to register this error code on _swagger_ @kistora

Reply:
Done